ng2-components/ng2-activiti-form/src/components/widgets/display-value/display-value.widget.spec.ts

```diff
@@ -0,0 +1,542 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Observable } from 'rxjs/Rx';
+import { DisplayValueWidget } from './display-value.widget';
+import { FormService } from '../../../services/form.service';
+import { FormFieldModel } from './../core/form-field.model';
+import { FormFieldTypes } from '../core/form-field-types';
+import { FormModel } from '../core/form.model';
+
+describe('DisplayValueWidget', () => {
+
+    let widget: DisplayValueWidget;
+    let formService: FormService;
+
+    beforeEach(() => {
+        formService = new FormService(null, null);
+        widget = new DisplayValueWidget(formService);
+    });
+
+    it('should require field to setup default value', () => {
+        widget.field = null;
+        widget.ngOnInit();
+        expect(widget.value).toBeUndefined();
+    });
+
+    it('should take field value on init', () => {
+        let value = '<value>';
+        widget.field = new FormFieldModel(null, { value: value });
```

Comment:
I was proposing moving the instances of `FormFieldModel` that you create into a separate file and import them into the test via a meaningful name. This will aid readability of the tests in this file and allow you over time to refactor the instances further to remove duplicated values or use helpers to construct them.